docs/UserGuide.md

```diff
@@ -87,11 +87,24 @@ Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
 
-### Listing all persons : `list`
+### Show all friends : `showfriends`
 
-Shows a list of all persons in the address book.
+Shows all friends in the address book. Switches GUI to the friends tab.
 
-Format: `list`
+Format: `showfriends`
+
+### Show a specific friend: `showfriend`
+
+Shows page containing the full details related to an existing friend in the address book. By default, only the latest 10 logs are displayed in this page.
+
+Format: `showfriend n/NAME -alllogs`
+
+* NAME field **must** be provided.
+* If the `-alllogs` flag is provided all the logs contained in that person will be displayed instead of the default 10 logs.
+
+Examples:
+* `showfriend n/John Doe` Will open up the page containing full details related to John Doe - his name, address, phone number, email, description, logs, and upcoming events with him.
+* `showfriend n/John Doe -alllogs` Will do the same as above except every single log will be displayed.
```

Comment:
I like how you described what the examples will do! We should probably standardize this across the board though. I was thinking that maybe this should be done only for the more complicated/less intuitive commands?

Reply:
Changed!